variables.tf

```diff
@@ -118,6 +118,12 @@ variable "ingress_prefix_list_ids" {
   default     = []
 }
 
+variable "ingress_with_prefix_list_ids" {
```

Comment:
There should be similar addition to `modules/_templates` to add support to named submodules.

Also, please update an example in `examples/http` to show the usage of this feature.

Reply:
Updated!